### DISCUSSION TITLE
Multiple 'Sync Errors' with Google Drive - are my files in sync or not?

### DISCUSSION THREAD

**Discussion by smilerish** [iterate-ch/cyberduck#16182] [Q&A]
I'm a daily user of Mountain Duck for mounting Google Drive on my Mac desktop. It generally works very well, and it's very useful.

However, there are lots of red 'Sync Error' icons scattered throughout my Google Drive mounts. The issue is, I can't find one place that neatly summarises the issues, and whether I need to worry about them!

I believe that most of these 'errors' are actually where a checksum is equal (so the local file and the remote file are identical - not really a sync error in my view!). But without sitting here going through each file one-by-one, how can I easily check the extent of these errors?

I've reached a stage where I want to rebuild my computer, so I want some assurance that my files safely in sync before I wipe my local system.

Any advice?

**Comment by tatze96** [comment]
The synchronization error badge icons indicate that an issue occurred during the synchronization. This causes the current version or the whole file, depending on the file action performed, to not be uploaded to the server.

You can generally try to resolve the synchronization errors by right-clicking on the affected item or its parent folder and choosing `Mountain Duck → Retry` from the appearing context menu.

If this doesn't resolve the synchronization errors, [enable debug logging](https://docs.cyberduck.io/mountainduck/support/#debug-log), recreate the synchronization error, and send us the log files with the ticket reference GH-16182 to support@mountainduck.io for further investigation.

**Reply by smilerish** [reply]
Thank you for the quick response.

I understand what the icons are for. The main point of my query was: is there a single place where I can see a list of all the errors? Otherwise, I have to navigate through the folders in the Finder looking for red error icons, which feels very inefficient.

Also, most of the errors state 'Unknown. Cancel upload for [filename] with equal checksum'. Presumably this means that the file doesn't need to by synced, but the error persists. If I click on 'Retry' then after attempting to sync, I get the same error message for that file.

I'll try the debugging suggestion to see if there's any more information available from the log.
